addons/web/static/src/core/tooltip/tooltip_service.js

```diff
@@ -39,17 +39,17 @@ import { whenReady } from "@odoo/owl";
  * with "info" being a stringified object with two keys "x" and "y".
  */
 
-const OPEN_DELAY = 400;
+export const OPEN_DELAY = 400;
 const CLOSE_DELAY = 200;
```

Comment:
Why not export also this constant (as we are on master) ?

Reply:
To have the minimal diff in case of back-port as this is a fix but ok I will do it.